tools/autograd/gen_variable_type.py

```diff
@@ -91,7 +91,7 @@
     'reflection_pad1d_backward', 'reflection_pad2d_backward',
     'replication_pad1d', 'replication_pad2d', 'replication_pad3d',
     'replication_pad1d_backward', 'replication_pad2d_backward', 'replication_pad3d_backward',
-    'diag', 'masked_scatter', 'masked_select', 'index_fill', 'trace', 'polar'
+    'diag', 'masked_scatter', 'masked_select', 'masked_fill', 'index_fill', 'trace', 'polar', 'unfold', 'unfold_backward'
```

Comment:
Your only change here should be adding `unfold` and `unfold_backward` to this list